divine/layout.py

```
@@ -0,0 +1,83 @@
+
+    def __str__(self):
+        return f'[{self.maxy}, {self.maxx}, {self.begy}, {self.begx}]'
+
+
+def validate_value(value):
+    # Covert the received value to Value() if received value was an 'int'
+    if isinstance(value, int):
+        value = Value(value)
+
+    # Covert the received value to Empty() if received value was a 'None'
+    elif value == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)